app/models/image.rb

```diff
@@ -1,2 +1,12 @@
 class Image < ActiveRecord::Base
+  has_many :items
+
+  has_attached_file :img,
+    :styles => { :menu_list => "150x150>", :item_list => "400x400>"  },
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Use the new Ruby 1.9 hash syntax.